spec/ruby/command_line/dash_v_spec.rb

```diff
@@ -6,8 +6,7 @@
 
   describe "when used alone" do
     it "prints version and ends" do
-      version = ruby_exe(nil, args: '--version')
-      ruby_exe(nil, args: '-v').should == version
+      ruby_exe(nil, args: '-v').include?(RUBY_DESCRIPTION).should == true
```

Comment:
`ruby -v` should only print the version imho, what's the use-case?

Reply:
GraalVM launchers have shared version printing method listing all languages and tools. So we print the GraalVM versions and then our full Ruby version string on the last line.

Reply:
Which launcher is used in that case?
I don't think `ruby -v` should print anything else than the ruby version (for `bin/polyglot` that's fine).

Reply:
What does it look like actually?

Reply:
The new aot launchers in GraalVM (new RubyLauncher class), I'll open PR with it soon. `ruby --polyglot` grants access to all the other languages so printing their versions makes sense in GraalVM. I don't have it build right now, please try `js --version` or `polyglot --version`. Unless there is something relying on the precise version format, which I doubt, I think unified user experience with GraalVM launchers is valuable.

Reply:
I could definitely imagine more than one thing depending on a single-line or sensible output for `ruby -v` or `ruby --version` though.

Output of GraalVM 0.27:
```
~/code/graalvm-0.27/bin ∴ ./js --version
GraalVM Polyglot Engine Version 0.27
  Installed Languages:
    JavaScript version 0.9
  Installed Tools:
    Agent            version 0.1
    CPU Sampler      version 1.0
    CPU Tracer       version 1.0
    Chrome Inspector version 0.1
    Memory Tracer    version 1.0

~/code/graalvm-0.27/bin ∴ ./polyglot --version 
GraalVM Polyglot Engine Version 0.27
  Installed Languages:
    JavaScript version 0.9
    llvm       version 0.01
  Installed Tools:
    Agent            version 0.1
    CPU Sampler      version 1.0
    CPU Tracer       version 1.0
    Chrome Inspector version 0.1
    Memory Tracer    version 1.0
```

Compared to other ruby engines' ruby -v this is very verbose.
```
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-linux]
```

It's also unclear how to the lines above will relate to the last line (our detailed version info).
But we can try it.

Reply:
fyi: our full version is printed at the bottom:
```
    ...
    Chrome Inspector version 0.1
    Memory Tracer    version 1.0

truffleruby unknown version, like ruby 2.3.3 <Java HotSpot(TM) 64-Bit Server VM 1.8.0_121-b13 without Graal> [darwin-x86_64]
```
It'll part of a different PR